app/javascript/components/Navigation/MatchLogger/HeadToHeadLogger.jsx

```diff
@@ -5,11 +5,10 @@ import PlayerSelect from '../PlayerSelect';
 const MatchLogger = ({ unlockedUsers, currentUserId }) => {
   const currentUserObject = unlockedUsers.find((user) => user.id === currentUserId);
   const sortedUsers = unlockedUsers.sort((a, b) => a.name.localeCompare(b.name));
-
   const [playerA, setPlayerA] = useState(currentUserObject);
   const [playerB, setPlayerB] = useState(currentUserObject);
   const [winnerId, setWinnerId] = useState();
-  const [selectedDate, setSelectedDate] = useState(new Date().toLocaleDateString('en-CA'));
```

Comment:
It seems strange that toLocaleDateString isn't working as expected anymore, and also weird that it was using en-CA instead of en-US (not that it should matter much)